Automatic Validation ended with:
> 2025-03-07T15:10:25.0437240Z ##[error] Manifest Error: Duplicate installer entry found.
 2025-03-07T15:10:25.0448779Z ##[error] Manifest Error: Duplicate installer entry found.

(Automated response - build 994.)

--- a/manifests/k/KRTirtho/Spotube/4.0.0/KRTirtho.Spotube.installer.yaml
+++ b/manifests/k/KRTirtho/Spotube/4.0.0/KRTirtho.Spotube.installer.yaml
@@ -24,6 +24,7 @@ Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/KRTirtho/spotube/releases/download/v4.0.0/Spotube-windows-x86_64-setup.exe
   InstallerSha256: 8834763B3687CEE813434B17E29379490725CAA20E259CEB1D842CB007DBFAF7
+  Scope: user
   InstallerSwitches:
     Custom: /CURRENTUSER
 ManifestType: installer

--- a/manifests/k/KRTirtho/Spotube/4.0.0/KRTirtho.Spotube.installer.yaml
+++ b/manifests/k/KRTirtho/Spotube/4.0.0/KRTirtho.Spotube.installer.yaml
@@ -7,7 +7,6 @@ InstallerLocale: en-US
 Platform:
 - Windows.Desktop
 InstallerType: inno
-Scope: machine
 ProductCode: 80B901C8-D6FE-494E-8AF7-A2BD440E8644_is1
 ReleaseDate: 2025-03-07
 AppsAndFeaturesEntries:

--- a/manifests/k/KRTirtho/Spotube/4.0.0/KRTirtho.Spotube.installer.yaml
+++ b/manifests/k/KRTirtho/Spotube/4.0.0/KRTirtho.Spotube.installer.yaml
@@ -18,6 +18,7 @@ Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/KRTirtho/spotube/releases/download/v4.0.0/Spotube-windows-x86_64-setup.exe
   InstallerSha256: 8834763B3687CEE813434B17E29379490725CAA20E259CEB1D842CB007DBFAF7
+  Scope: machine
   InstallerSwitches:
     Custom: /ALLUSERS
 - Architecture: x64

